src/gui/status_footer.cpp

```diff
@@ -49,7 +49,10 @@ void status_footer_t::windowEvent(EventLock /*has private ctor*/, window_t *send
 
     if (dynamic_cast<screen_home_data_t *>(GetParent()) != nullptr //is home_screen
         && sheet_number_of_calibrated() > 1) {                     // calibrated more profiles than 1
-        update_sheet_profile();
+        if (mseconds - last_timer_repaint_profile >= REPAINT_PROFILE_PERIOD) {
```

Comment:
OMG, this is brutal - that means, we are repainting the footer too fast ... so basically any form of slowing-down like you just did for the sheet will be benefical.